sequelizeDatabase/sequelFunctions.js

```diff
@@ -1,350 +1,481 @@
-    }).catch(err => console.error(err));
+  return models.Individuals.findAll()
+    .then(individuals => {
+      return individuals;
+    })
+    .catch(err => console.error(err));
 };
 getAllCompanies = () => {
-    return models.Companies.findAll().then((companies) => {
-        return companies;
-    }).catch(err => console.error(err));
+  return models.Companies.findAll()
+    .then(companies => {
+      return companies;
+    })
+    .catch(err => console.error(err));
 };
 getAllFunds = () => {
-    return models.Funds.findAll().then((funds) => {
-        return funds;
-    }).catch(err => console.error(err));
+  return models.Funds.findAll()
+    .then(funds => {
+      return funds;
+    })
+    .catch(err => console.error(err));
 };
 getAllEmployeeHistory = () => {
-    return models.EmployeeHistory.findAll().then((employeeHistory) => {
-        return employeeHistory;
-    }).catch(err => console.error(err));
+  return models.EmployeeHistory.findAll()
+    .then(employeeHistory => {
+      return employeeHistory;
+    })
+    .catch(err => console.error(err));
 };
 
 getAllOriginalFundPositions = () => {
-    return models.OriginalFundPosition.findAll().then((originalPositions) => {
-        return originalPositions;
-    }).catch(err => console.error(err));
-}
+  return models.OriginalFundPosition.findAll()
+    .then(originalPositions => {
+      return originalPositions;
+    })
+    .catch(err => console.error(err));
+};
 
 getAllSharedFunds = () => {
-    return models.SharedFunds.findAll().then((sharedFunds) => {
-        return sharedFunds;
-    }).catch(err => console.error(err));
-}
+  return models.SharedFunds.findAll()
+    .then(sharedFunds => {
+      return sharedFunds;
+    })
+    .catch(err => console.error(err));
+};
 
 //--------------------Insert into Table Functions-----------//
 
-insertUser = (Username, Password, Email) =>{
-    return models.User.findOrCreate({where:{Email: Email},
-        defaults: {
-            Username: Username,
-            Password: Password
-        }
-    }).spread((user, createdBoolean) =>{
-        return user;
-    }).catch(err => console.error('Error in "insertUser", ', err));
+insertUser = (Username, Password, Email) => {
+  return models.User.findOrCreate({
+    where: { Email: Email },
+    defaults: {
+      Username: Username,
+      Password: Password
+    }
+  })
+    .spread((user, createdBoolean) => {
+      return user;
+    })
+    .catch(err => console.error('Error in "insertUser", ', err));
 };
 
 insertFund = (FundName, UserID) => {
-    return models.Funds.create({ //TODO: Make this a findOrCreate to prevent duplicate names.
-        FundName: FundName,
-        UserID: UserID
-    }).then((createdFund) => {
-        return createdFund;
-    }).catch(err => console.error('Error in "insertFund", ', err));
+  return models.Funds.create({
+    //TODO: Make this a findOrCreate to prevent duplicate names.
+    FundName: FundName,
+    UserID: UserID
+  })
```

Comment:
**ISSUE**: Make this so that a single user cannot have two or more funds with the same name.